eta/detectors/tfmodels_detectors.py

```diff
@@ -385,6 +382,157 @@ def _evaluate(self, imgs, ops):
         return self._sess.run(out_tensors, feed_dict={in_tensor: imgs})
 
 
+class TF2ModelsDetectorConfig(Config, etal.HasPublishedModel):
+    """TF2ModelsDetector configuration settings.
+
+    Note that `labels_path` is passed through
+    `eta.core.utils.fill_config_patterns` at load time, so it can contain
+    patterns to be resolved.
+
+    Attributes:
+        model_name: the name of the published model to load. If this value is
+            provided, `model_path` does not need to be
+        model_path: the path to a frozen inference graph to load. If this value
+            is provided, `model_name` does not need to be
+        labels_path: the path to the labels map for the model
+        confidence_thresh: a confidence threshold to apply to candidate
+            detections
+    """
+
+    def __init__(self, d):
+        d = self.init(d)
+
+        self.labels_path = etau.fill_config_patterns(
+            self.parse_string(d, "labels_path")
+        )
+        self.pipeline_path = etau.fill_config_patterns(
```

Comment:
`pipeline_path` appears to be unused here

Reply:
Thought I deleted that, removed from the FiftyOne `manifest.json` as well.